message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -111,6 +111,16 @@
         <param index="6">Empty.</param>
         <param index="7">Empty.</param>
       </entry>
+      <entry value="42006" name="MAV_CMD_FIXED_MAG_CAL_YAW">
+        <description>Magnetometer calibration based on provided known yaw. This allows for fast calibration using WMM field tables in the vehicle, given only the known yaw of the vehicle. If Latitude and longitude are both zero then use the current vehicle location.</description>
+        <param index="1">YawDegrees.</param>
+        <param index="2">CompassMask, 0 for all.</param>
+        <param index="3">Latitude degrees.</param>
```

Comment:
Is there some reason not to use the traditional fields 5 and 6 here?

I realise the precision isn't important (if `COMMAND_INT` is used), but I can't see the hurt in trying to be uniform.

Reply:
@peterbarker i don't really want this to be filled in in most cases. zero/zero is the normal value. I don't want someone to send a "target position" without reading and understanding this msg